apps/payments/e2e/fixtures/test.ts

```
@@ -1,9 +1,13 @@
 import { test as baseTest, expect } from "@playwright/test";
 import fs from "fs";
 import path from "path";
+import { LoginPage } from "../objects/loginPage";
+import { password, publicServantUser } from "../utils/constants";
 
-const baseURL = process.env.HOST_URL ?? "http://localhost:3001";
-const loginUrl = process.env.LOGIN_URL ?? "http://localhost:8000/static/login/";
+const baseURL =
```

non-blocking: do we know if there is any way to define the env vars we need with playwright with a json schema like in Fastify? I'd personally prefer our tests to fail instead of having a fallback that might be wrong depending on the environment

---

As far as I can see the env vars are injected if you enable the devServer config. If you run the application you want to test in a separate process, as I'm doing now, you have no env vars at all. It's something that we can further investigate in a separate PR :)